modules/admin/templates/config.twig

```diff
@@ -18,7 +18,7 @@
     <div class="message-box">
       {% trans %}SimpleSAMLphp is installed in:{% endtrans %}
       <kbd>{{ directory }}</kbd><br/>
-      {% trans %}You are running version <kbd>{{ version }}</kbd>.{% endtrans %}
+      {% trans %}You are running version <kbd>%version%}</kbd>.{% endtrans %}
```

Comment:
Stray `}` here. And is %version% automatically replaced somehow?

Reply:
No that is an oversight on my part - will push a fix in a few minutes.